x-pack/plugins/task_manager/server/config.ts

```diff
@@ -51,6 +51,14 @@ const eventLoopDelaySchema = schema.object({
   }),
 });
 
+const taskConfig = schema.object({
+  skip: schema.object({
+    enabled: schema.boolean({ defaultValue: true }),
```

Comment:
**question:** should we have this disabled by default to begin with?
```suggestion
    enabled: schema.boolean({ defaultValue: false }),
```

Reply:
yes eventually i'll disable it but before that want to be sure that all the tests are still running when enabled.